packages/numeric-input/src/NumericInput.js

```diff
@@ -0,0 +1,178 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { ThemeContext } from "@hig/theme-context";
+
+import { css } from "emotion";
+import Input from "@hig/input";
+
+import customStylesheet from "./customStylesheet";
+import ControlBehavior from "@hig/behaviors/src/ControlBehavior";
```

Comment:
You can import `ControlBehavior` from `@hig/behaviors` w/o having to go further down the `src` directory. I think the `imports` would make more sense to have all external dependencies together, then `@hig` componenets, then the relative `imports`.